src/Microsoft.DotNet.Interactive/KernelInvocationContextExtensions.cs

```diff
@@ -35,6 +35,30 @@ public static class KernelInvocationContextExtensions
 
             return displayedValue;
         }
+        
+        public static DisplayedValue DisplayAs(
+            this KernelInvocationContext context,
+            string value,
+            string mimeType = null)
+        {
+           
+            var displayId = Guid.NewGuid().ToString();
+
+            var formattedValue = new FormattedValue(
+                mimeType,
```

Comment:
should use here the preferred mimeType if the one passed is null or whitespaces?